packages/ag-charts-community/src/scene/group.ts

```diff
@@ -14,115 +14,46 @@ export class Group extends Node {
         return value instanceof Group;
     }
 
-    private clipRect?: BBox;
-    protected layer?: HdpiCanvas;
+    static computeChildrenBBox(nodes: Iterable<Node>, skipInvisible = true) {
+        function* visible() {
+            for (const n of nodes) {
+                if (!skipInvisible || (n.visible && !n.transitionOut)) {
+                    const bbox = n.getBBox();
+                    if (bbox) yield bbox;
+                }
+            }
+        }
```

Comment:
Nit(micro-optimisation): Should we extract this as a top-level function to improve optimisability?